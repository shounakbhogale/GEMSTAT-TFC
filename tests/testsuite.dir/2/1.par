{"tfs":{"foo":{"annot_thresh":0.6,"maxbind":10.7981,"alpha_a":4.51529,"alpha_r":1}},"inter":{},"qbtm":[0.04072],"enh":[{"pi":1,"beta":2.5558}]}
