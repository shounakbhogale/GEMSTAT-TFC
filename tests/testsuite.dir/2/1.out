Rows	1	2
test_GT	1	0.1	test	1	0.1
